### DISCUSSION TITLE
How to change row color on hover?

### DISCUSSION THREAD

**Discussion by Klaus-8** [KevinVandy/material-react-table#672] [Q&A] [answered]
I'm trying to change row color on mouse hover.
I have tried : 

`muiTableBodyRowProps={({ row }) => ({
              sx: {
                '&:hover': {
                  backgroundColor: '#e8f1fd'
                }
              }
            })}`

But this doesn't worked. 
Is their any other way to achieve it?

**Comment by jrmalin** [comment, marked as answer]
Try using `'&:hover td'`, so `muiTableBodyRowProps={({ row }) => ({ sx: { '&:hover td': { backgroundColor: '#e8f1fd' } } })}`!

**Reply by davidloh85** [reply]
I did that, but if you enable renderDetailPanel, the hover effect appears onto the container holding the reactNode element that is being displayed as well. 

I guess the container was treated as a <td> as well.
